dradis/dradis-ce
# Default opened tab

### Specs
This is specially true when destroying. Users sometimes want keep selectively deleting evidence, and after each deletion they have to go click again on the evidence tab.

**Proposed solution** 
Add a way in JS to open a default tab when a URL has #<asdf>-tab much like how anchors work.

### How to test:

1. Create an issue and a node
2. Visit the issue and open the evidence tab
3. Create multiple evidences and confirm that you are routed back with the evidence tab opened.
4. Update an evidence and confirm that you are routed back with the evidence tab opened.
5. Delete the evidence and confirm that you are routed back with the evidence tab opened.

### Check List
